src/stdlib_math_logspace.fypp

```diff
@@ -0,0 +1,64 @@
+#:include "common.fypp"
+#:set RANKS = range(1, MAXRANK + 1)
+#:set RC_KINDS_TYPES = REAL_KINDS_TYPES + CMPLX_KINDS_TYPES
+#:set IR_KINDS_TYPES = INT_KINDS_TYPES + REAL_KINDS_TYPES
```

Comment:
```suggestion
```

Reply:
Removed unused variables in latest commit